### ISSUE TITLE
Find in terminal not work 

### ISSUE THREAD

**Issue by ErvinXie** [microsoft/vscode#170531]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.74.2
- Commit: e8a3071ea4344d9d48ef8a4df2c097372b0c5161
- OS Version: Darwin x64 21.1.0

Steps to Reproduce:

1. launch terminal in vscode, focus on it. 
2. cmd+f, the search box appears on the editor, not the terminal.

**Comment by meganrogge** [comment]
I cannot reproduce, pls attach a gif

**Comment by meganrogge** [comment]
thanks for that. pls use this command to see what's going on with your keybindings
![Screenshot 2023-01-04 at 8 39 57 AM](https://user-images.githubusercontent.com/29464607/210579349-985ee849-799e-419b-be30-bc67fe9c2db5.png)

**Comment by ErvinXie** [comment]
![截屏2023-01-04 22 57 05](https://user-images.githubusercontent.com/29942298/210583298-8f9b5cfd-e4f7-40bb-979e-b699b3c139b4.png)
This is what i find after press cmd+f with focus on terminal. I guess there is something wrong with the JetBrains IDE Keymap extension with detailed info below. I deactivated the extension and finding block reappears, and I activated the extension again and there is problem again. 

Name: JetBrains IDE Keymap
ID: isudox.vscode-jetbrains-keybindings
Version: 0.1.9
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=isudox.vscode-jetbrains-keybindings

Thanks so much for your helping!
